test/mocks/router/mocks.h

```diff
@@ -271,6 +282,7 @@ class MockRouteEntry : public RouteEntry {
   std::multimap<std::string, std::string> opaque_config_;
   TestVirtualCluster virtual_cluster_;
   TestRetryPolicy retry_policy_;
+  TestHedgePolicy hedge_policy_;
```

Comment:
You're missing an `ON_CALL` in the `MockRouteEntry` constructor to wire this up to the mocked function